### ISSUE TITLE
Banner ads interfere with the interface

### ISSUE THREAD

**Issue by inside83** [kslimani/clappr-ima-plugin#34]
Hi
I'm using Google's video ads and if they serve pre-roll, it's all fine.
But if it's a banner, like on the screenshot below, closing an ad (which then minimizes itself) or even if it's up, the interface is unresponsive.
Viewer is unable to click on anything but play/pause, including volume, cast, fullscreen...
There's no error in the console or anything which would point in the right direction.
![image](https://user-images.githubusercontent.com/10822610/106067111-46c1d580-60fe-11eb-9a82-20faca1134cb.png)
![image](https://user-images.githubusercontent.com/10822610/106067126-4cb7b680-60fe-11eb-9403-04d99330ca90.png)

**Comment by kslimani** [comment]
There is a known issue. ([issue #3](https://github.com/kslimani/clappr-ima-plugin/issues/3)). It should happen mainly on iPhone.

**Comment by inside83** [comment]
Thank you for your swift reply.
I can confirm it's happening on Win10/Chrome and on TizenOS's browser.
I'm using clappr/player v0.4.0 since newer versions don't support casting and 0.3.x has issues with respecting `playback.hlsjsConfig.maxMaxBufferLength` value.

**Comment by kslimani** [comment]
The plugin have no option to disable all non-linear ads, thought it is a good idea (and very simple to implement), i will take a look.

**Comment by kslimani** [comment]
I released a new version with `disableNonLinear` plugin configuration option.

**Comment by inside83** [comment]
wow, that was super fast 😀👍
is it on jsdelivr too?
`https://cdn.jsdelivr.net/npm/clappr-ima-plugin@latest/dist/clappr-ima-plugin.min.js`

**Comment by kslimani** [comment]
Yes, it should be available on jsDelivr (ie: cache should be purged and it should be available as latest version).

**Comment by inside83** [comment]
Just a quick follow up:
You said the plugin isn't fully compatible with Clappr >= 0.4.x.
What's the latest Clappr version the plugin is fully compatible and are there any plans on making it compatible to Clappr >= 0.4.x?

**Comment by kslimani** [comment]
This plugin should work with latest 0.3.x version. (i think it is 0.3.13)

Clappr 0.3.x jsDelivr url : https://cdn.jsdelivr.net/npm/clappr@latest/dist/clappr.min.js

_Note: Clappr 0.4.x has many breaking changes. It is one of the reason it is distributed on another npm namespace (@clappr).
Clappr 0.4.x jsDelivr url : https://cdn.jsdelivr.net/npm/@clappr/player@latest/dist/clappr.min.js_

I have no exact ETA to making this plugin compatible with 0.4.x but it is definitively in my roadmap.
There are still many changes pending (especially with playbacks internal plugins).
I am currently monitoring these changes and waiting for 0.4 to reach some stability before making this plugin compatible.
